test/widget/crowd_league_app/crowd_league_app_test.dart

```diff
@@ -124,9 +124,7 @@ void main() {
       firebaseCompleter.complete();
 
       // Setup a mock store and complete the redux future with the store.
-      final middleware = VerifyDispatchMiddleware();
-      final store = Store<AppState>(appReducer,
-          initialState: AppState.init(), middleware: [middleware]);
+      final store = Store<AppState>(appReducer, initialState: AppState.init());
```

Comment:
Just removed the middleware (that no longer calls it's NextDispatcher) so that actions will propagate through to reducers and (potentially) change the state.

Reply:
sweet